CHANGELOG.md

```diff
@@ -3,6 +3,25 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## [8.4.0] Stable Release
+### Added
+- Added support for Data Classification v2 [#1338](https://github.com/microsoft/mssql-jdbc/pull/1338)
+- Added public API to return the database compatibility level [#1345](https://github.com/microsoft/mssql-jdbc/pull/1345)
+- Added support for Azure SQL DNS Caching [#1357](https://github.com/microsoft/mssql-jdbc/pull/1357) [#1373](https://github.com/microsoft/mssql-jdbc/pull/1373)
+
+### Fixed issues
+- Fixed an issue with fetching column information provides wrong order of columns intermittently [#1348](https://github.com/microsoft/mssql-jdbc/pull/1348)
+- Fixed an issue with spatial datatypes casting error when Always Encrypted is enabled [#1353](https://github.com/microsoft/mssql-jdbc/pull/1353)
+- Fixed an issue with getColumns() not returning correct type for IS_AUTOINCREMENT and IS_GENERATEDCOLUMN against Azure Data Warehouse [#1356](https://github.com/microsoft/mssql-jdbc/pull/1356)
+- Fixed an issue with STAsBinary returning WKB format instead of CLR format [#1364](https://github.com/microsoft/mssql-jdbc/pull/1364)
+- Fixed an issue with allowing non-MSSQL ResultSets to bulk copy DateTimeOffset [#1365](https://github.com/microsoft/mssql-jdbc/pull/1365)
+- Fixed issues identified by SonarQube [#1369](https://github.com/microsoft/mssql-jdbc/pull/1369)
+- Fixed an issue with batch insertion failing when Always Encrypted is enabled [#1378](https://github.com/microsoft/mssql-jdbc/pull/1378)
+
```

Comment:
what about executeBatch issue 1334?

Reply:
1334 is a pull request and it got closed without being merged, so I didn't put it in here.

Reply:
sorry, I meant to say 1378, which is already there, was confused..